src/components/Match/MatchStory.jsx

```diff
@@ -156,6 +156,32 @@ const formatList = (items, noneValue = []) => {
   }
 };
 
+// evaluate the sentiment behind the message - rage, question, statement etc
+const evaluateSentiment = (event, lastMessage) => {
+  const { message, player, time } = event;
+  const sentiment = message.indexOf('?') !== -1 ? ['question'] : ['statement'];
+
+  if (lastMessage && lastMessage.time + 130 > time) {
+    if (lastMessage.player_slot === player.player_slot) {
+      sentiment.push('continued');
+    } else if (lastMessage.key.indexOf('?') !== -1) {
```

Comment:
Also making explicit: looks like someone can't "respond" to another message unless that other message is a question, right? Again, I think this is fine; just making sure it's intended. 👍

Reply:
Yes, was intended. Thinking can split it out into 2 separate locale's - responses to questions, and responses in general. At first it was getting too many responses without the question check, but then I added in the time check which seems to cover most scenario's.